activerecord/lib/active_record/associations/builder/collection_association.rb

```diff
@@ -66,17 +66,5 @@ def #{name.to_s.singularize}_ids=(ids)
         end
       CODE
     end
-
-    def self.wrap_scope(scope, mod)
```

Comment:
Ah, I understood what you meant. Actually this changes the following result:

```ruby
post_reflection = Post.reflect_on_association(:comments)
p post_reflection.options # => {:extend=>[PostCommentsAssociationExtension]}
```

But `HasManyReflection ` is `:nodoc:` class.
https://github.com/rails/rails/blob/v5.1.0.rc2/activerecord/lib/active_record/reflection.rb#L681

If we should not change the behavior, I'll change this PR to keep the previous behavior.